### DISCUSSION TITLE
Cannot render colab jupyter notebooks while logged in my account

### DISCUSSION THREAD

**Discussion by pjmartel** [community/community#59319] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Feature Area

Issues

### Body

While logged in my account I cannot render a Colab Jupyter notebooks. If I sign out of my account, 
or log into a different account, the notebooks display fine. 

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @pjmartel ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
